### ISSUE TITLE
Network.set_options doesn't work when loading the graph from a DOT file.

### ISSUE THREAD

**Issue by tuschla** [WestHealth/pyvis#129]
I tried using the `set_options` method as [boludo00 demonstrated](https://github.com/WestHealth/pyvis/issues/30#issuecomment-509345610). This works fine. 

However when I read the edges and nodes from a DOT file the options I set afterwards with `set_options` have no effect.  

What you can do to replicate this:
```py
import pyvis
graph = pyvis.network.Network()
graph.from_DOT("./example_dotfile.dot")
graph.set_options("""
var options = {
    "physics": {
        "enabled": false,
        "stabilization": 10,
        "barnesHut": {
            "springLength": 200
        }
    }
}
""")
graph.show("test.html")
```
   
Here is an example DOT file for testing:
```DOT
digraph {
    a [label=12];
    b [label=7];
    c [label=3];
    a -> b[label="solid edge"];                     
    a -> b [label="dashed edge"]; 
    a -> c [label="dashed edge"]; 
    a -> c [label="dotted edge"];      
}
```

Adding the following to the generated js will work though.
```js
options.physics = {
    enabled: false,
    stabilization: 10,
    barnesHut: {
        springLength: 200
    }
}
```

**Comment by Darkproduct** [comment]
Here is the fix:
https://github.com/WestHealth/pyvis/blob/b11d105db7059dab1a2c511a740868c54f415bc7/pyvis/templates/template.html#L165-L168

Change those lines to:
```js
var options = parsedData.options;
options = Object.assign(options, {
    nodes: {
        shape: "dot"
    },
});
{% if options %}
options = Object.assign(options, {{options|safe}})
{% endif %}
```

**Comment by tuschla** [comment]
I can confirm this works! :)

**Comment by jhunpingco** [comment]
@Darkproduct Can you write a pull request for this change?

**Comment by Darkproduct** [comment]
Yea, I can do that. Should this also include the fix for #128? I mean the version of [`vis-network`](https://github.com/visjs/vis-network) this project currently uses is 5? years old.

**Comment by jhunpingco** [comment]
Please. That makes sense.
